IBAnimatable/ActivityIndicatorAnimationBallRotateChase.swift

```diff
@@ -30,9 +30,8 @@ public class ActivityIndicatorAnimationBallRotateChase: ActivityIndicatorAnimati
 
 
 // MARK: - Setup
-
+// swiftlint:disable variable_name_min_length
```

Comment:
It is basically the same. Make a pull request if you think it is better. 
Cause right now it is the only thing preventing me from build on Xcode 8 with swiftlint